lib/fog/metering/openstack/requests/get_samples.rb

```diff
@@ -2,7 +2,7 @@ module Fog
   module Metering
     class OpenStack
       class Real
-        def get_samples(meter_id, options = [])
+        def get_samples(meter_id, options = [], limit = 10000000)
```

Comment:
Wouldn't it be possible to avoid changing the interface using `options = [:limit => 10000000]` instead?